mkdocs.yml

```
@@ -27,6 +27,11 @@ nav:
     - Build system: build-system.md
     - Communicate with JavaScript: communicate-with-javascript.md
     - Melange for X developers: melange-for-x-developers.md
+  - Melange for React Developers:
+    - Introduction: melange-book/index.md
```

Should we adapt this path to be 
```suggestion
    - Introduction: melange-for-react-developers/index.md
```

---

Actually, I was kinda hoping to shorten the URL further by just making it 

```suggestion
    - Introduction: book/index.md
```

Wdyt?

---

ping

---

I think we will be publishing the book initially in the `unstable` branch of the docs site, so I'm good with `book/`.